playground/src/containers/LifecycleScreen.js

```diff
@@ -3,7 +3,7 @@ const { Component } = require('react');
 
 const { View, Text, Button } = require('react-native');
 
-const Navigation = require('react-native-navigation');
+const { Navigation } = require('react-native-navigation');
```

Comment:
why the destructuring? this should stay a simple export

Reply:
it's because of exporting not only Navigation. It can be changed after improving issue with export